python/samples/concepts/plugins/crew_ai_plugin.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import logging
+
+from semantic_kernel import Kernel
+from semantic_kernel.core_plugins.crew_ai import CrewAIEnterprise
+from semantic_kernel.core_plugins.crew_ai.crew_ai_models import InputMetadata
+from semantic_kernel.functions import KernelArguments, KernelFunction
+
+logging.basicConfig(level=logging.INFO)
+
+
+async def using_crew_ai_enterprise():
+    # Create an instance of the CrewAI Enterprise Crew
+    crew = CrewAIEnterprise()
+
+    #####################################################################
+    #              Using the CrewAI Enterprise Crew directly            #
+    #####################################################################
+
+    # The required inputs for the Crew must be known in advance. This example is modeled after the
```

Comment:
Is there any other README or getting started info required for this? I have never used CrewAI before, especially an Enterprise Crew. If I had, would this stuff makes sense to me?